pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -245,33 +246,33 @@ func dryRunPreemption(ctx context.Context, fh framework.PreemptHandle, state *fr
 				Pods:             pods,
 				NumPDBViolations: int64(numPDBViolations),
 			}
-			s := strategy{
-				victims:           &victims,
-				nominatedNodeName: nodeInfoCopy.Node().Name,
+			c := candidate{
+				victims: &victims,
+				name:    nodeInfoCopy.Node().Name,
 			}
-			strategies = append(strategies, &s)
+			candidates = append(candidates, &c)
 			resultLock.Unlock()
 		}
 	}
 	parallelize.Until(ctx, len(potentialNodes), checkNode)
-	return strategies
+	return candidates
 }
 
-// CallExtenders call given <extenders> to filter out <strategies>, and return a filtered strategies.
-// We will only check <strategies> with extenders that support preemption.
+// CallPreemptExtenders call given <extenders> to filter out <candidates>, and return a filtered candidates.
+// We will only check <candidates> with extenders that support preemption.
 // Extenders which do not support preemption may later prevent preemptor from being scheduled on the nominated
 // node. In that case, scheduler will find a different host for the preemptor in subsequent scheduling cycles.
-func CallExtenders(extenders []framework.Extender, pod *v1.Pod, nodeLister framework.NodeInfoLister,
-	strategies []Strategy) ([]Strategy, error) {
+func CallPreemptExtenders(extenders []framework.Extender, pod *v1.Pod, nodeLister framework.NodeInfoLister,
```

Comment:
I disagree with the name change, I think "`CallPreemptExtenders`" doesn't really read well, also this function is in the preemption plugin, so there is enough context to know what `CallExtenders` mean.

Reply:
How about `FindCandidatesWithExtenders` similar to the previous name ？:smirk:

Reply:
I think `FindCandidatesWithExtenders` is a bit verbose... I'd like to revert to `CallExtenders`.